style.css

```diff
@@ -0,0 +1,3 @@
+form {
+  display: flex;
+}
```

Comment:
## OPTIONAL CHANGES 

- [ ] While this is not a hard requirement, it would be great to have a  styled UI for great User Experience and it would also help with having a good screenshot to have in the project's README